nighres/registration/embedded_syn.py

```diff
@@ -0,0 +1,235 @@
+# basic dependencies
+import os
+import sys
+
+# main dependencies: numpy, nibabel
+import numpy as np
+import nibabel as nb
+
+# for external tools: nipype
+from nipype.interfaces.ants import ANTS
+# using the global interface rather than specific ones
+#from nipype.interfaces.ants import WarpImageMultiTransform
+#from nipype.interfaces.ants import WarpTimeSeriesImageMultiTransform
+from nipype.interfaces.ants import ApplyTransforms
+
+# cbstools and nighres functions
+import cbstools
+from ..io import load_volume, save_volume
+from ..utils import _output_dir_4saving, _fname_4saving, \
+                    _check_topology_lut_dir
+
+# convenience labels
+X=0
+Y=1
+Z=2
+T=3
+
+def embedded_syn(source_image, target_image, coarse_iterations=40, medium_iterations=50, fine_iterations=40,
+					run_affine_first=False, cost_function='MutualInformation', interpolation='NearestNeighbor',
+                    save_data=False, output_dir=None,
+                    file_name=None):
+    """ Embedded SyN
+
+    Runs the Symmetric Normalization (SyN) algorithm of ANTs and formats the output deformations
+    into voxel coordinate mappings as used in CBSTools registration and transformation routines.
```

Comment:
We should add that this requires installation of Nipype and ANTs

Reply:
I could remove the Nipype layer if needed (in the end it doesn't bring much in this case)

Reply:
Ok sounds good, less dependencies